ftp_server/pyftpdlib/servers.py

```diff
@@ -266,13 +266,15 @@ def handle_accepted(self, sock, addr):
                 if ip is not None and ip in self.ip_map:
                     self.ip_map.remove(ip)
 
-    def handle_error(self):
-        """Called to handle any uncaught exceptions."""
-        try:
-            raise
-        except Exception:
-            logger.error(traceback.format_exc())
-        self.close()
+    # Commented this out as it overrides a parent class and the 'raise'
+    #
+    # def handle_error(self):
+    #     """Called to handle any uncaught exceptions."""
+    #     try:
+    #         raise
+    #     except Exception:
+    #         logger.error(traceback.format_exc())
+    #     self.close()
```

Comment:
You are using a versioning control system (VCS), don't comment out code, delete it, it will be available in the file's history.